src/UI/Component/DropDown/DropDownItem.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/* Copyright (c) 2017 Alexander Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\DropDown;
+
+use ILIAS\UI\Component\Clickable;
+use ILIAS\UI\Component\Component;
+use ILIAS\UI\Component\Hoverable;
+use ILIAS\UI\Component\JavaScriptBindable;
+
+/**
+ * Drop down items
+ */
+interface DropDownItem extends Component, JavaScriptBindable, Clickable, Hoverable {
```

Comment:
Hi @alex40724, @Amstutz,

@nhaagen is currently working on the breadcrumb, and, surprise, also found the concept of a link. I.e. the breadcrumbs are nothing more than a specialized list of links, very similar to the dropdown in that regard. So maybe a "link list" could be a node in the hierarchy, with the children "dropdown" and "breadcrumb".
@nhaagen also, imo correctly, observed, that a "link" (i.e. a tuple of URL and title) seems to be a data type in its own right, independent from the UI framework, as it should also be usable in other contexts. That means we might want to introduce it in src/Data instead of the UI-framework. This would also solve the context problem. What do you think? We are able to propose an implementation for Data now, if we agree to that solution.
Regards!